dinky-cdc/dinky-cdc-plus/src/main/java/org/dinky/cdc/SinkBuilderFactory.java

```diff
@@ -24,35 +24,29 @@
 import org.dinky.cdc.doris.DorisSchemaEvolutionSinkBuilder;
 import org.dinky.cdc.doris.DorisSinkBuilder;
 import org.dinky.cdc.kafka.KafkaSinkBuilder;
-import org.dinky.cdc.kafka.KafkaSinkJsonBuilder;
 import org.dinky.cdc.sql.SQLSinkBuilder;
-import org.dinky.cdc.starrocks.StarrocksSinkBuilder;
+import org.dinky.cdc.sql.catalog.SQLCatalogSinkBuilder;
 import org.dinky.exception.FlinkClientException;
 import org.dinky.model.FlinkCDCConfig;
 
 import java.util.HashMap;
 import java.util.Map;
 import java.util.function.Supplier;
 
-/**
- * SinkBuilderFactory
- *
- * @since 2022/4/12 21:12
- */
 public class SinkBuilderFactory {
 
     private static final Map<String, Supplier<SinkBuilder>> SINK_BUILDER_MAP =
             new HashMap<String, Supplier<SinkBuilder>>() {
+
                 {
+                    put(SQLSinkBuilder.KEY_WORD, () -> new SQLSinkBuilder());
+                    put(SQLCatalogSinkBuilder.KEY_WORD, () -> new SQLCatalogSinkBuilder());
                     put(KafkaSinkBuilder.KEY_WORD, () -> new KafkaSinkBuilder());
```

Comment:
private static final Map<String, Supplier<SinkBuilder>> SINK_BUILDER_MAP = ImmutableMap.of(
            SQLSinkBuilder.KEY_WORD, SQLSinkBuilder::new,
            SQLCatalogSinkBuilder.KEY_WORD, SQLCatalogSinkBuilder::new,
            KafkaSinkBuilder.KEY_WORD, KafkaSinkBuilder::new,
            DorisSinkBuilder.KEY_WORD, DorisSinkBuilder::new,
            DorisExtendSinkBuilder.KEY_WORD, DorisExtendSinkBuilder::new,
            DorisSchemaEvolutionSinkBuilder.KEY_WORD,
            DorisSchemaEvolutionSinkBuilder::new
    );